server/registry/server_test.go

```diff
@@ -43,20 +43,28 @@ func defaultTestServer(t *testing.T) *RegistryServer {
 	t.Helper()
 
 	if !usePostgres {
-		return serverWithSQLite(t)
+		if server, err := serverWithSQLite(t); err != nil {
+			t.Fatalf("Setup: failed to get server with SQLite: %s", err)
```

Comment:
Since we're not actually handling this error, can we fail the test inside the `serverWithSQLite` function? The Postgres function returns an error so we can handle it by falling back to SQLite - here it just complicates the control flow and duplicates our fail code.

Reply:
Yes, this code is more verbose - but OTOH I think it would be odd for serverWithSQLite() and serverWithPostgres() to treat internal errors differently only to optimize the way that these functions are used in this instance. I'd rather err on the side of consistency here.